tchannel/tornado/connection.py

```diff
@@ -49,6 +51,13 @@
 log = logging.getLogger('tchannel')
 
 
+#: Sentinel object representing that the connection is outgoing.
+OUTGOING = object()
```

Comment:
Yeah. Consider,

    foo(True, False, True)

Versus,

    foo(Incoming, LazyReads, Ephemeral)

With named arguments we can add readability to the former case (`foo(incoming=True, strict_reads=False, ephemeral=True)`) but then sentinel values allow for more than two states (true/false). In this case it doesn't matter much since we'll only ever have two directions, except that maybe sentinel values are slightly more informative (`direction=incoming` vs `incoming=True`).